lib/spack/docs/pipelines.rst

```diff
@@ -594,28 +744,33 @@ environment variables used by the pipeline infrastructure are described here.
 AWS_ACCESS_KEY_ID
 ^^^^^^^^^^^^^^^^^
 
-Needed when binary mirror is an S3 bucket.
+Optional.  Only needed when binary mirror is an S3 bucket.
```

Comment:
Those are double spaces, is there a convention about that in `rst`?  Should we just insert a single space after a period?

Reply:
I could've sworn I counted 3, but apparently I've learned how to count since 10am this morning. That looks fine (I don't like the double-space after a period, but I know I'm on the losing side of that).